addons/binding/org.openhab.binding.airquality/src/main/java/org/openhab/binding/airquality/internal/json/AirQualityJsonTime.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.airquality.internal.json;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+
+/**
+ * The {@link AirQualityJsonTime} is responsible for storing
+ * the "time" node from the waqi.org JSON response
+ *
+ * @author Kuba Wolanin - Initial contribution
+ */
+public class AirQualityJsonTime {
+
+    private String s;
+    private String tz;
+
+    private final static SimpleDateFormat SDF = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+
+    /**
+     * Get Timezone in following format: "+0100"
+     *
+     * @return {String}
+     */
+    public String getTz() {
+        return tz.replace(":", "");
```

Comment:
I'm looking at this on GitHub only, so sorry if I missed that: Where it `tz` initialized?